client/src/app/(modules)/practices/(main)/(details)/field.tsx

```diff
@@ -25,32 +26,66 @@ const Field = ({ label, value, url, hasEllipsis, logo, formatId }: FieldType) =>
     setIsExpanded(!isExpanded);
   };
 
-  const renderLink = (url: string | string[]) =>
-    Array.isArray(url) ? (
-      <div>
-        {url.map(
-          (u, i) =>
-            value?.[i] && (
-              <>
-                {i !== 0 ? ', ' : ''}
-                <a
-                  key={u}
-                  href={u}
-                  target="_blank"
-                  rel="noreferrer"
-                  className="text-sm text-brown-500"
```

Comment:
@Bluesmile82 can you review and confirm which classNames we should be using in all cases? It seems like we were using 2 different classes, not sure why.

Reply:
For practices it always should be text-brown-500 @yulia-bel 
I think we are always using the array link. But just in case we need another link let's keep it.